modules/blake3/1.3.3/presubmit.yml

```diff
@@ -0,0 +1,14 @@
+matrix:
+  platform:
+  - centos7
```

Comment:
The centos run fails due to defaulting to an old C standard. It's a very outdated runner, so you could also just drop it.

Reply:
Done.